en/payloads/README.md

```diff
@@ -19,50 +19,80 @@ The MAVSDK provides simple APIs to use this protocol for both [standalone camera
+See [Camera triggering](../peripherals/camera.md) for information on how to configure your camera to work with MAVLink.
 
-Grippers can also be triggering using the [MAV_CMD_DO_GRIPPER](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_GRIPPER) MAVLink command, or manually via a Joystick button.
+:::note
+Cameras connected directly to the flight controller _only_ support camera triggering, and are unlikely to be suitable for most surveillance/search work.
+:::
 
-For setup and usage information see:
+A search and rescue drone may also need to carry cargo, for example, emergency supplies for a stranded hiker.
+See [Cargo Drones](#cargo-drones-package-delivery) above for information about payload delivery.
 
-- [Gripper](../peripherals/gripper.md)
-- [Flying > Package Delivery Mission Planning](../flying/package_delivery_mission.md)
+## Agricultural Drones/Crop Spraying
 
-### Generic Actuator Control with RC
+Agricultural drones are commonly used for mapping crop health and pest detection and animal management (herding, tracking, etc.).
+These cases are similar to the [mapping](#mapping-drones) and [surveillance, search & rescue](#surveillance-search-rescue) cases above.
+While specific crops/animals may need specialist cameras, the integration with PX4 is the same.
 
-Up to 6 autopilot PWM or CAN outputs can be controlled using RC channels.
-The outputs that are to be controlled are specified in the [Actuators](../config/actuators.md#actuator-outputs) configuration screen by assigning the functions `RC AUX 1` to `RC AUX 6` to the desired [actuator outputs](../config/actuators.md#actuator-outputs).
+Agricultural drone may also be used for crop spraying.
+In this case the sprayer must be controlled as a [generic actuator](#generic-actuator-control):
 
-To map a particular RC channel to an output function `RC AUX n` (and hence it's assigned output) you use the [RC_MAP_AUXn](../advanced_config/parameter_reference.md#RC_MAP_AUX1) parameter that has the same `n` number.
+- Most sprayers provide controls to activate/deactivate a pump; some also allow control over the rate of flow or the spray field (i.e. by controlling the nozzle shape, or using a spinner to distribute the payload).
+- The [Generic Actuator Control with MAVLink Command](#generic-actuator-control-with-mavlink-command) section explains how you can connect flight controller outputs to your sprayer and trigger them using the `MAV_CMD_DO_SET_ACTUATOR`.
+- You can define the area to spray using a [Survey pattern](https://docs.qgroundcontrol.com/master/en/PlanView/pattern_survey.html), or you can define the grid to fly using waypoints.
+  In either case, it is important to ensure that the vehicle flight path provides adqequate coverage for your payload.
+- You should add a `MAV_CMD_DO_SET_ACTUATOR` to your mission before and after the survey pattern in order to enable and disable the sprayer.
```

Comment:
Does this command actually work? Would be good thing to verify it with a MAVSDK example...

Reply:
@julianoes I'm going to assume that they do. Firstly because @junwoo091400 says they should, secondly because there is the MAVSDK example below that I understand was used to test this.

Reply:
And lastly, because the code to support this is present. It should have been tested when that went in. This section is already present in docs - I've just moved it around. So retesting would be quite a bit of work for me, and we're no worse off if I do not.